### PR TITLE
Latest NLS translations

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile amac jdk17

**Comment by keithc-ca** [comment]
```
[2024-09-24T20:19:15.281Z] * Failure occured during the process *
[2024-09-24T20:19:15.281Z] Error: J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1 Message "??????????%zu????? %d ???????????%zu????" in /Users/jenkins/workspace/Build_JDK17_aarch64_mac_Personal/openj9/runtime/nls/shrc/j9shr_zh_CN.nls is different from "The page size (%zu) of the operating system is different from the page size (%zu) of the system on which layer %d cache is created." in /Users/jenkins/workspace/Build_JDK17_aarch64_mac_Personal/openj9/runtime/nls/shrc/j9shr.nls
```

**Comment by keithc-ca** [comment]
It looks like translation permuted the placeholders, but didn't qualify them with indices (e.g. `%1$zx`).

**Comment by keithc-ca** [comment]
It appears that the messages must specify argument indices if any language needs to insert the values in another order:
```
J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=The page size (%1$zu) of the operating system is different from the page size (%2$zu) of the system on which layer %3$d cache is created.
```

**Comment by pshipton** [comment]
Jenkins compile amac jdk17

**Comment by pshipton** [comment]
Updated and compiling now.

File: runtime/nls/shrc/j9shr_ca.nls [hunk 1]
```diff
@@ -6913,3 +6908,36 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.explanation=Th
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=The JVM continues.
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
+
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=La mida de p\u00e0gina ( %1$zu ) del sistema operatiu \u00e9s diferent de la mida de p\u00e0gina ( %2$zu ) del sistema en qu\u00e8 es crea la mem\u00f2ria cau de la capa %3$d .
```
Comment (keithc-ca) [inline runtime/nls/shrc/j9shr_ca.nls:6912]: Did the automation add the spaces inside the parentheses? I don't think we want them. They're not added universally, e.g. they're not added for `de`, `fr` or `it`.

Comment (pshipton) [reply runtime/nls/shrc/j9shr_ca.nls:6912]: Yes the automation added them. We could remove them manually but then would always have to fix this up when doing future translations, which I don't want to do. I suggest we just accept them, they aren't harmful.

Comment (keithc-ca) [reply runtime/nls/shrc/j9shr_ca.nls:6912]: This is disappointing. There are other places in this same file (e.g. line 6293) where those spaces were not added, so we can't say it's a custom of certain languages. I think you're right that they aren't harmful, but I don't know for sure. Can you follow-up with the maintainers of the translation tool to see whether inconsistencies like this are addressed?

Comment (pshipton) [reply runtime/nls/shrc/j9shr_ca.nls:6912]: Good news, I discovered how to manually update translations, see https://github.com/eclipse-openj9/openj9/pull/20230

**Pull request merged**